pointcloud_to_grid_node:
  ros__parameters:
    pointcloud_to_grid:
      topic_name:
        map: "/map"
        cloud: "/glim_ros/points"
        # cloud: "/livox/lidar"
      tf_frame:
        map_frame: "map"
      add_cell: true
      filter:
        voxelgrid_size: 0.01
        z_max: 1.0
        z_min: 0.1
      grid_map_info:
        resolution: 0.1 #[m] def 0.05
        # origin: [-5.0,-5.0,0.0]
        width: 25.0  #[m]
        height: 25.0 #[m]
